src/test/java/org/folio/finc/select/FincSelectFiltersIT.java

```diff
@@ -268,4 +268,30 @@ public void checkThatWeCannotCreateFiltersToCollectionAssociationIfFilterNotPres
         .then()
         .statusCode(400);
   }
+
+  @Test
+  public void checkThatWeReturnAnEmptyListAnd200IfFilterWithNoCollectionIsQueried() {
+    filter1.setId(UUID.randomUUID().toString());
+
+    given()
+        .body(Json.encode(filter1))
+        .header("X-Okapi-Tenant", TENANT_UBL)
+        .header("content-type", ContentType.JSON)
+        .header("accept", ContentType.JSON)
+        .post(FINC_SELECT_FILTERS_ENDPOINT)
```

Comment:
Given the existing test design, created resources should be properly cleaned up to avoid interfering with other test cases.